tests/server/test_tracing_api.py

```diff
@@ -0,0 +1,54 @@
+import time
+import pytest
+from bentoml import config
+
+
+@pytest.mark.skipif(
+    not config("tracing").get("opentracing_server_address"),
```

Comment:
how do we set this config in the CI tests environment?

Reply:
how do you configure Zipkin tests in the CI tests, it should practically the same